src/problem/MFEMProblem.C

```diff
@@ -516,48 +520,58 @@ MFEMProblem::setMOOSEElementalVarData(MooseVariableFieldBase & moose_var_ref)
 }
 
 InputParameters
-MFEMProblem::addMFEMFESpaceFromMOOSEVariable(InputParameters & moosevar_params)
+MFEMProblem::addMFEMFESpaceFromMOOSEVariable(InputParameters & parameters)
```

Comment:
We'll probably remove this, since we'll now only be creating `MFEMVariables` in `MFEMProblem`. This logic would then move into checks in future Transfer objects between MFEM and MOOSE variables